mfai/torch/models/half_unet.py

```diff
@@ -61,7 +64,7 @@ def forward(self, x):
         return self.relu(x)
 
 
-class HalfUNet(ModelABC, nn.Module):
+class HalfUNet(ModelABC, nn.Module, AutoPaddingModel):
```

Comment:
For the sake of clarity I would put the two ABCs first:

```suggestion
class HalfUNet(ModelABC, AutoPaddingModel), nn.Module):
```